test/src/server/routes/entity/entity.js

```diff
@@ -80,3 +84,391 @@ describe('getDefaultAliasIndex', () => {
+
+		let workBRelationshipSet = await new RelationshipSet({id: workBRelationshipSetId})
+			.save(null, {method: 'insert'})
+			.then(
+				(model) =>
+					model.relationships().attach([relationshipB]).then(() => model)
+			);
+		workBRelationshipSet = await workBRelationshipSet.refresh({withRelated: 'relationships'});
+
+		// Set the Entity Attributes as Required
+		author.set('revisionId', revisionId);
+		author.set('relationshipSetId', authorRelationshipSet.get('id'));
+		workA.set('revisionId', revisionId);
+		workA.set('relationshipSetId', workARelationshipSet.get('id'));
+		workB.set('revisionId', revisionId);
+		workB.set('relationshipSetId', workBRelationshipSet.get('id'));
+
+		// Save the Entities and Update the database
+		await author.save(null, {method: 'update'});
+		await workA.save(null, {method: 'update'});
+		await workB.save(null, {method: 'update'});
+
+		// Get updated Entity
+		workA = await new Work({bbid: workA.get('bbid')}).fetch({withRelated: 'relationshipSet.relationships'});
+
+		// Get list of Affected Entities
+		const affectedEntities = await deleteRelationships(orm, null, workA.toJSON());
+
+		const affectedEntityRelationshipSet = await new RelationshipSet({
+			id: affectedEntities[0].get('relationshipSetId')
+		}).fetch({withRelated: 'relationships'});
+
+		expect(affectedEntities[0].get('bbid')).to.equal(author.get('bbid'));
+		expect(affectedEntityRelationshipSet.toJSON().relationships[0].targetBbid).to.be.equal(workB.get('bbid'));
+		expect(affectedEntities).to.have.length(1);
+	});
+	it('it should return an array with multiple other entities to modify if there are relationships to multiple entities', async () => {
+		// Create Entites
+		const workABBID = getRandomUUID();
+		const workBBBID = getRandomUUID();
+		const workCBBID = getRandomUUID();
+		const workA = await createWork(workABBID);
+		const workB = await createWork(workBBBID);
+		const workC = await createWork(workCBBID);
+
+		// Create Revision
+		const revisionId = random.number();
+		await new Revision({authorId: editorJSON.id, id: revisionId})
+			.save(null, {method: 'insert'});
+
+		// Create Relations between entity
+		const relationshipDataA = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: workA.toJSON().bbid,
+			typeId: 1
+		};
+		const relationshipDataB = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: workB.toJSON().bbid,
+			typeId: 1
+		};
+		const relationshipDataC = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: workC.toJSON().bbid,
+			typeId: 1
+		};
+
+		// Relate Relationship Data with Relationship Type
+		relationshipDataA.typeId = relationshipTypeData.id;
+		relationshipDataA.id = random.number();
+
+		relationshipDataB.typeId = relationshipTypeData.id;
+		relationshipDataB.id = random.number();
+
+		relationshipDataC.typeId = relationshipTypeData.id;
+		relationshipDataC.id = random.number();
```

Comment:
Why not set those directly in the objects above?
Just a suggestion to try and reduce the number of lines.
(this could apply to the other tests as well)